integration-cli/docker_utils.go

```diff
@@ -458,6 +460,32 @@ func (d *Daemon) sock() string {
 	return fmt.Sprintf("unix://%s/docker.sock", d.folder)
 }
 
+func (d *Daemon) getBaseDeviceSize(c *check.C) int64 {
+
```

Comment:
can you maybe clean this up a bit, random carrige returns and vars should not be starting with a capital